internal/storage/mem/mem.go

```diff
@@ -108,18 +108,29 @@ func (s *Storage) Restore() error {
 }
 
 func (s *Storage) Backup() error {
+	if s.opt == nil || s.opt.BackupPath == "" || s.opt.StoreInterval == 0 {
```

Comment:
```suggestion
	if s.opt == nil || s.opt.BackupPath == "" || s.opt.StoreInterval > 0 {
```
Функция должна срабатывать при включенном режиме синхронизации (`s.opt.StoreInterval == 0`) и должна НЕ срабатывать при s.opt.StoreInterval > 0

Reply:
готово